remappings.txt

```diff
@@ -0,0 +1 @@
+@openzeppelin/=lib/openzeppelin-contracts/
```

Comment:
please remove remapping, we don't need this for libraries, it can lead to path resolution error if being imported from parent directory.